source/tutorials/first-steps/declarative-shell.md

```diff
@@ -32,7 +32,7 @@ In this tutorial we'll take a look how to create reproducible shell environments
 
 ## Entering a shell with Python installed
 
-Suppose we a development environment in which Python 3 was installed.
+Suppose we want a development environment in which `git, neovim and nodejs` was installed.
```

Comment:
Either use the project name (in their own spelling and capitalisation) or the program name.

```suggestion
Suppose we want a development environment in which Git, Neovim, and Node.js were installed.
```

```suggestion
Suppose we want a development environment in which `git`, `nvim`, and `node` were installed.
```